packages/documentation/docs/controls/_category-filter_styleguide.md

```diff
@@ -0,0 +1,71 @@
+The category filter is a user interface component that allows users to narrow down a large set of data based on predefined categories. 
+As users start typing in the input field, the filter displays matching categories in a dropdown, which can be selected easily with autocomplete. 
+Users can also specify a condition, such as equals (=), not equals (!=), bigger (>), smaller(<), or equals "fixed word", to further refine their search. 
+After selecting a category and a condition, users enter the filter term, which can be either suggested via autocomplete or entered as free text. The category filter provides an efficient and user-friendly way to browse and switch between different categories. It's customizable and can be adapted to the specific needs of your application or website.
+
+It allows for efficient browsing and users can easily switch between different categories.
+
+![Category filter overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=1799%3A38402&mode=design&t=hgAA8GogE70JbHHy-1)
+1. Container 
+2. Search icon 
+3. Input chip 
+4. Clear button  
+
+## Options
+
+- **Categories**: These are the predefined groups that users can select to narrow down their search or browsing. The categories are customizable and should be defined based on the specific needs of your application or website.
+- **LabelCategories**: ???
```

Comment:
these are the labels (aka the string) shown for the categories

Reply:
in general, since this is UX focused, you can use a normal writing style here (e.g. instead of LabelCategories -> category labels)

Reply:
i18nLabelCategories.. not relevant for UX